### PR TITLE
Fixing the formatting issues

### PR DESCRIPTION
**Pull request by Pensu**
Fixes https://github.com/openebs/openebs/issues/1973

There were some other formatting issues other than the ones in pkg/storagepool/storagepool_template_engine.go, this PR fixes them as well.

### CODE REVIEW

File: pkg/storagepool/storagepool_template_engine.go [hunk 1]
```diff
@@ -19,11 +19,11 @@ package storagepool
 
 import (
 	"fmt"
-	"strings"
 	"github.com/ghodss/yaml"
 	"github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
 	"github.com/openebs/maya/pkg/engine"
 	"github.com/openebs/maya/pkg/util"
+	"strings"
 )
```
Comment (prateekpandey14) [inline pkg/storagepool/storagepool_template_engine.go]: Same as above comment

Comment (Pensu) [reply pkg/storagepool/storagepool_template_engine.go]: @prateekpandey14 I just ran ```go fmt``` on the said file and got the above result. I don't think there would be any other change in that particular file in terms of formatting, I will revert these changes, but then is the original issue valid?

Comment (Pensu) [reply pkg/storagepool/storagepool_template_engine.go]: So I ran ```goimport``` and revert the changes, but now Better Code Hub is panicking, any idea why?

Comment (Pensu) [reply pkg/storagepool/storagepool_template_engine.go]: Ok, got it, doesn't look like the files need any changes, closing this PR. Apologies for the noise. Probably the original issue should be closed as well.

**Pull request closed without merging**